### ISSUE TITLE
[java] UseTryWithResources does not handle multiple argument close methods

### ISSUE THREAD

**Issue by JJengility** [pmd/pmd#1701]
**Affects PMD Version:** 6.12

**Rule:** UseTryWithResources 

**Description:** We have a custom close method that we use closeLocalResources that can take in multiple parameters to close them. PMD is not flagging this as an issue when there are multiple parameters however it does successfully flag the issue when there is only 1 parameter in this custom close method.

**Code Sample demonstrating the issue:**

```
closeLocalResources(ps, rs); is not flagged while

closeLocalResources(ps); is flagged
```

**Running PMD through:** *[Ant]*

**Comment by jsotuyod** [comment]
We can probably drop the `[@ArgumentCount = 1]` condition from the XPath

**Comment by jainrish** [comment]
I would like to take this up. Since it will be my first contribution, let me know if there's anything that will be helpful.

**Comment by adangel** [comment]
That's great, be welcomed to PMD!

First step is setting up a development environment. You can follow the steps here: https://github.com/pmd/pmd/wiki/Newcomers%27-Guide and https://github.com/pmd/pmd/wiki/Setup-IDE

For this issue, you should then add an additional test case, which reproduces the problem. The test cases are here: https://github.com/pmd/pmd/blob/master/pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/UseTryWithResources.xml

And then you can fix the rule. The rule is defined in the best practices ruleset: https://github.com/pmd/pmd/blob/master/pmd-java/src/main/resources/category/java/bestpractices.xml#L1522

Let me know, if you have any question.

**Comment by Han-Cui** [comment]
Should the label of this issue false-positive instead of false-negative? When encountering multiple parameters, PMD doesn't warn as expected.
